Install_Scripts/install_mactex.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "Installing MikTeX with Homebrew"
+brew cask install mactex
```

Comment:
Brew is not installed by default on Mac machines.  It comes bundled with XCode command line tools.  Might want to do something similar to what we do for the windows install or make a note of it in pre-reqs if that is something we want to live with.

Reply:
I replaced this script with putting this command in the readme text. From my experience it is completely fine to assume a Mac developer has Homebrew installed.